src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheckTest.java

```diff
@@ -162,6 +162,16 @@ public void testLiteralDo() throws Exception {
                 expected);
     }
 
+    @Test
+    public void testLiteralYield() throws Exception {
+        final String[] expected = {
+            "16:9: " + getCheckMessage(MSG_WS_NOT_FOLLOWED, "yield"),
+        };
+        verifyWithInlineConfigParser(
+                getPath("InputWhitespaceAfterLiteralYield.java"),
```

Comment:
There is method getnoncompiledpath is something like this.
Please use it

Reply:
Done